tests/microbenchmarks/test_bench_moe_routing_control.py

```
@@ -0,0 +1,1067 @@
+    for src in range(4):
+        ids, _ = bench_moe._materialize_selected_experts_for_rank(
+            plan,
+            src_rank=src,
+            top_k=4,
+            experts_per_rank=4,
+            moe_ep_size=4,
+            device=torch.device("cpu"),
+            scale_dtype=torch.float32,
+        )
+        per_rank_ids.append(ids)
+    obs_slot, obs_token, obs_hist = bench_moe._observe_routing_metrics(
```

Ruff (0.15.13)

[warning] Unpacked variable `obs_hist` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)